test/test-hooks-sample-warning.js

```diff
@@ -92,7 +92,7 @@ describe('express + dbs', function() {
         http.get({port: common.serverPort + 1}, function(res) {
           server.close();
           common.cleanTraces(agent);
-          assert.equal(debugCount, 2);
+          assert.equal(debugCount, 0);
```

Comment:
As discussed in person, we should remove dead code from this test on the log checks and open a bug to track plugin specific log messages.